core-feature-pack/src/main/resources/modules/system/layers/base/org/wildfly/embedded/main/module.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2010, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<module xmlns="urn:jboss:module:1.3" name="org.wildfly.embedded">
+    <properties>
+        <!-- Core version of the embedded API is currently only meant
+        for use by the CLI -->
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <exports>
+        <exclude path="org/jboss/as/embedded/logging"/>
+    </exports>
+
+    <resources>
+        <artifact name="${org.wildfly.core:wildfly-embedded}"/>
+    </resources>
+
+    <dependencies>
+        <module name="org.wildfly.security.manager"/>
+        <module name="org.jboss.as.server"/>
+        <module name="org.jboss.as.controller"/>
+        <module name="org.jboss.as.controller-client"/>
+        <!--module name="org.jboss.as.naming"/-->
+        <module name="org.jboss.jandex"/>
+        <module name="org.jboss.logging"/>
+        <module name="org.jboss.modules"/>
+        <module name="org.jboss.staxmapper"/>
+        <module name="org.jboss.stdio"/>
+        <module name="org.jboss.vfs"/>
+        <module name="org.jboss.as.protocol"/>
+    </dependencies>
```

Comment:
There should probably be a dependency on `org.jboss.logmanager` as well. Possibly optional as it's optional in the pom.